docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/hemoglobin/index.rst

```diff
@@ -6,4 +6,237 @@ Hemoglobin Model
 
 .. contents::
    :local:
-   :depth: 2
\ No newline at end of file
+   :depth: 2
+
+Hemoglobin Model Description in GBD 2019
+----------------------------------------
+
+The hemoglobin model in GBD 2019 serves as the underlying basis for both the anemia impairment model and iron deficiency risk factor model. The hemoglobin model output is an estimate of the year-, location-, age-, and sex-speciifc **continuous ensemble distribution of population hemoglobin concentration in grams per liter**.
+
+The hemoglobin distribution is estimated from a variety of sources reported as either anemia prevalence and/or mean and standard deviation hemoglobin concentration; altitude adjustments were made when appropriate and possible, although no smoking adjustments were performed. For data sources that *only* included pregnant women, data were crosswalked to the general population using MR-BRT such that pregnant women were matched to non-pregnant women in the same age and location group and the ratios of their hemoglobin concentrations were assessed. The resulting adjustment factor was 0.92 (95% CI: 0.86 - 0.98), such that the hemoglobin level of pregnant women is 0.92 times that of non-pregnant women.
+
+The hemoglobin distribution was modeled in three steps:
+
+1. ST-GPR models of mean and standard deviation hemoglobin concentration (including pregnancy adjustments as described above). 
+
+    Covariates for the mean model included Age-specific Fertility Rate, HIV Prevalence, SEV for Child underweight, SEV for Child wasting, Malaria Incidence, Haemoglobin C (sickle type C) trait (all ages), Haemoglobin S (sickle type S) trait (all ages), Sociodemographic Index, SEV for Impaired kidney function, Healthcare Access and Quality index, Modern contraception prevalence, and 50th percentile of haemoglobin (pooled across all microdata sources). 
+
+    Covariates for the standard deviation model included: Malaria Incidence, Haemoglobin C (sickle type C) trait, Haemoglobin S (sickle type S) trait, Sociodemographic Index, SEV for Impaired kidney function, Healthcare Access and Quality index, Education Relative Inequality (Gini), 50th percentile of haemoglobin (pooled across all microdata sources), and mean haemoglobin (results from mean [Hb] ST-GPR model)
+
+2. Calculation of ensemble weights
+
+    A set of two-parameter distributions (gamma, mirror gamma, Weibull, mirror lognormal, and mirror gumbel) were fit to the sample’s haemoglobin mean and variance for each location/year/age/sex group
+
+    The weights used for the GBD 2019 hemoglobin distribution model were 40% gamma and 60% mirror gumbel.
+
```

Comment:
I think I understand how the ensemble weights work because I've seen them before, but I don't think this provides enough clarity for an unfamiliar reader.

Here's my understanding of the steps for finding an ensemble distribution (correct me if I'm wrong):
1. Each of the two-parameter distributions is "fit" to the data in each location/year/age/sex group using the "[method of moments](https://en.wikipedia.org/wiki/Method_of_moments_(statistics))." I.e. you just choose the distribution that has the same mean and standard deviation as the data in each group, so there's not actually any complicated curve fitting going on here.

2. The distribution F of hemoglobin is assumed to be a [mixture distribution](https://en.wikipedia.org/wiki/Mixture_distribution) that also comes from a 2-parameter family (GBD calls this an ensemble), of the form
F(x; m_i,s_i) = w_1 F_1(x; m_i,s_i) + ... + w_k F_k(x; m_i,s_i),
where
m_i = mean Hb in group i
s_i = standard deviation of Hb in group i
F_1,...,F_k are the distributions in the list above (gamma, mirror gamma,etc.), parameterized by mean m and standard deviation s
F is the mixture distribution for Hb, also parameterized by mean m and standard deviation s
w_1,...,w_k are the **global** weights of the distributions in the mixture, used in all groups
(I think the important thing to note here is that the distribution weights are *global*, not depending on the location/year/age/sex, whereas it is only the mean and standard deviation that vary across groups. This is not totally clear from your description above.)

3. The global weights w_1,...,w_k are found by solving an optimization problem that somehow chooses the "best" weights that match the data in all the groups simultaneously. According to the YLD appendix, this "best fit" is defined by taking anemia prevalence data into account.  Whatever they did to solve this optimization problem, the weights they came up with are w_1 = 0.4 (for F_1 = gamma distribution) and w_2 = 0.6 (for F_2 = mirror gumbel distribution), with w_j = 0 for all the other distributions.

Reply:
Yes, this is consistent with my understanding. I will add this detail to the existing documentation

Reply:
Cool. It's probably not necessary to give all the detail I wrote above, as most of that probably belongs on a separate page about ensemble distributions in GBD.  But I think we should at least write down the formula for what the hemoglobin distribution actually is, e.g.

F(x | mu,sigma) = 0.4 F_1(x | mu,sigma) + 0.6 F_2(x | mu,sigma),

where
F_1(x | mu,sigma) = gamma distribution parameterized by mean mu and standard deviation sigma,
F_2(x | mu,sigma) = mirror Gumbel distribution parameterized by mean mu and standard deviation sigma,
and mu and sigma depend on the location/year/age/sex (and pregnancy status).

(By "distribution" I mean "cumulative distribution function," but the same formulas also hold for the probability density functions.)

And ideally we would write down the mathematical formulas for the gamma and mirror Gumbel distributions, but probably don't worry about that yet since we still aren't sure what the correct formula for the mirror Gumbel distribution is.

Reply:
I added a description of the distribution and also added your walk through of ensemble distribution as a note that we could eventually move elsewhere but provides good context here for now.